rewrite/rewrite/python/style.py

```diff
@@ -347,6 +347,30 @@ def keep_indents_on_empty_lines(self) -> bool:
     def with_keep_indents_on_empty_lines(self, keep_indents_on_empty_lines: bool) -> TabsAndIndentsStyle:
         return self if keep_indents_on_empty_lines is self._keep_indents_on_empty_lines else replace(self, _keep_indents_on_empty_lines=keep_indents_on_empty_lines)
 
+    @dataclass(frozen=True)
+    class MethodDeclarationParameters:
+        _align_multiline_parameters: bool
+
+        @property
+        def align_multiline_parameters(self) -> bool:
+            return self._align_multiline_parameters
+
+        def with_align_multiline_parameters(self,
+                                            align_multiline_parameters: bool) -> TabsAndIndentsStyle.MethodDeclarationParameters:
+            return self if align_multiline_parameters is self._align_multiline_parameters else replace(self,
+                                                                                                       _align_multiline_parameters=align_multiline_parameters)
+
+    _method_declaration_parameters: MethodDeclarationParameters
```

Comment:
We need to make the same modification to the Java side, where we declare this style.

Reply:
Added